Emby.Server.Implementations/SocketSharp/WebSocketSharpRequest.cs

```diff
@@ -335,21 +309,21 @@ public string PathInfo
                 {
                     var mode = HandlerFactoryPath;
 
-                    var pos = request.RawUrl.IndexOf('?', StringComparison.Ordinal);
+                    var pos = RawUrl.IndexOf("?", StringComparison.Ordinal);
```

Comment:
I wonder, I would assume ASP.NET Core has a built in way to split HTTP URLs apart.

Reply:
There's QueryHelper.ParseQuery or something like that, but I'd rather not change too much. I've already had to fix a couple of bugs caused by differing behaviours between the custom and the asp.net implementations.